lib/new_relic/agent/instrumentation/redis/middleware.rb

```diff
@@ -0,0 +1,16 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module RedisClient
+    module Middleware
+      # This module is used to instrument Redis 5.x+
```

Comment:
I like the `+` here. Could you please update the comments in `instrumentation.rb` and `prepend.rb` to use it as well for consistency?

Reply:
Sure thing! Resolved in: d89e3b40145355970ba4c546df76cd12707e96c0